### PR TITLE
Completed US500 TC002

### PR DESCRIPTION
**Pull request by SF001User**
Finished by adding awesome cucumber stuff here

### CODE REVIEW

**Comment by TegzoK** [comment]
perfect, merge

**Review by TegzoK — APPROVED** [review]

**Review by lo-guliko — CHANGES_REQUESTED** [review]
please format it ))

File: src/test/java/b22user/US500TC002.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package b22user;
+
+public class US500TC002 {
+    // work has started with bunch of cucumber stuff
+    // SALAD IS DONE!!
```
Comment (aliakpa) [inline src/test/java/b22user/US500TC002.java:5]: Perfection!

**Review by johnnyone-github — APPROVED** [review]
looks good

File: src/test/java/b22user/US500TC002.java (hunk 1 above)
Comment (IrinaPK8) [reply src/test/java/b22user/US500TC002.java:5]: great work!

**Pull request merged**